apps/hbase/src/hbase/templates/app.mako

```diff
@@ -2515,7 +2515,7 @@ ${ commonheader(None, "hbase", user, request) | n,unicode }
       $(this).trigger('reset');
     });
 
-    var prepareNewTableForm = function () {
+    window.prepareNewTableForm = function () {
```

Comment:
Would this work?

function prepareNewTableForm()

Reply:
I modified it to "function prepareNewTableForm()" and tested, it has the same error and doesn't work.
prepareNewTableForm function is inside of $(document).ready(){}， it is invisible outside only if in the form of 'window.prepareNewTableForm'.